tools/dump_kubernetes.sh

```diff
@@ -164,12 +163,14 @@ tap_containers() {
       for container in ${containers}; do
 
         for f in "${functions[@]}"; do
-          "${f}" "${namespace}" "${pod}" "${container}"
+          "${f}" "${namespace}" "${pod}" "${container}" | return $?
```

Comment:
I think you mean to use `||`, not a pipe (`|`).

Reply:
Please also add to the comments above `tap_containers` (line 147) that it exits as soon as one of the functions fails.